docs/usage/configuration-options.md

```diff
@@ -363,10 +363,13 @@ The "topic" is usually refers to the dependency being updated, e.g. `"dependency
 
 ## composerIgnorePlatformReqs
 
-By default, Renovate will run Composer with `--ignore-platform-reqs` as the PHP platform used by Renovate most probably won't match with the required PHP environment of your project as configured in your `composer.json` file.
-However, this also means that all platform constraints (including PHP version) will be ignored by default, which can result in updated dependencies that are not compatible with your platform.
+By default, Renovate will ignore Composer platform requirements as the PHP platform used by Renovate most probably won't match with the required PHP environment of your project as configured in your `composer.json` file.
```

Comment:
```suggestion
By default, Renovate will ignore Composer platform requirements as the PHP platform used by Renovate most probably won't match the required PHP environment of your project as configured in your `composer.json` file.
```

Reply:
Is this still true for binarySource=docker though?

Reply:
> Is this still true for binarySource=docker though?

AFAIK know this does not influence `composerIgnorePlatformReqs` which is `[]` by default. Which leads to `--ignore-platform-reqs` being added by default (to ignore ext/lib platform reqs), which makes composer also ignore the `php` platform req.